Formula/rav1e.rb

```diff
@@ -5,12 +5,13 @@ class Rav1e < Formula
   head "https://github.com/xiph/rav1e.git", branch: "master"
 
   stable do
-    url "https://github.com/xiph/rav1e/archive/v0.6.1.tar.gz"
-    sha256 "dd12132ad9dac229ce00a9caad132c4ad23d7db2b3ad4b5a59e89658fee04d9a"
+    url "https://github.com/xiph/rav1e/archive/v0.6.2.tar.gz"
+    sha256 "8fe8d80bc80a05ee33113c0ee19779d9c57189e5434c8e1da8f67832461aa089"
 
+    # keep the version in sync
     resource "Cargo.lock" do
```

Comment:
I think this would be an excelent one for @nandahkrishna their recent livecheck improvements